package.json

```diff
@@ -29,8 +29,9 @@
     "gulp"
   ],
   "dependencies": {
-    "gulp-util": "~2.2.14",
-    "through2": "~0.6.3"
+    "plugin-error": "0.1.2",
+    "through2": "2.0.3",
+    "vinyl": "2.1.0"
```

Comment:
`vinyl`  should be in the `devDependencies`

Reply:
How can vinyl be in devDependencies when it's used by the code? Did you mean that it should be in peer-dependencies?

Reply:
I only see it used in `test.js` but I may be missing something.

Reply:
Yup, my bad, you're right

Reply:
Could you fix it and push the change?

Reply:
Thank you! I will fix it.